src/java.base/share/classes/java/util/zip/ZipInputStream.java

```diff
@@ -25,9 +25,10 @@
 
 package java.util.zip;
 
-import java.io.InputStream;
-import java.io.IOException;
 import java.io.EOFException;
+import java.io.IOException;
+import java.io.InputStream;
```

Comment:
Nitpick: should the `InputStream` import precede that of `IOException`?

Reply:
> Nitpick: should the `InputStream` import precede that of `IOException`?

Gonna leave this as is as IntelliJ keeps changing it and not worth fighting